src/helpers.ts

```diff
@@ -9,6 +9,7 @@ import {
     TubularHttpClient,
     Transformer,
 } from 'tubular-common';
+import camelcase from 'camelcase'
```

Comment:
This is totally an implementation-specific requirement. We should not add camelcase package just to format the exported file. That can be done by the implementation.